mrchecker-framework-modules/mrchecker-webapi-module/src/test/java/com/capgemini/mrchecker/webapi/httpbin/HtmlLinksTest.java

```diff
@@ -0,0 +1,95 @@
+package com.capgemini.mrchecker.webapi.httpbin;
+
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.capgemini.mrchecker.webapi.BasePageWebApiTest;
+import com.capgemini.mrchecker.webapi.pages.httbin.LinksPage;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+import org.junit.Test;
+
+import java.text.MessageFormat;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class HtmlLinksTest extends BasePageWebApiTest {
+	private LinksPage linksPage = new LinksPage();
+
+	@Test
+	public void notFoundOnNegativeNParam() {
+		int n = -1;
+		int offset = 5;
+		validateNegativeParam(n, offset);
+	}
+
+	@Test
+	public void notFoundOnNegativeOffsetParam() {
+		int n = 5;
+		int offset = -1;
+		validateNegativeParam(n, offset);
+	}
+
+	private void validateNegativeParam(int n, int offset) {
```

Comment:
Name of method do not correlate with implementation - 'validate', but GET is sending.
Please consider to use Parameters for this tests.
Example from https://www.testwithspring.com/lesson/writing-parameterized-tests-with-junit-4/:
```
@RunWith(JUnitParamsRunner.class)
@Category(UnitTest.class)
public class JUnitParamsCalculatorTest {
 
    @Test
    @Parameters({
            "0, 0, 0",
            "1, 1, 2"
    })
    public void shouldReturnCorrectSum(int first, 
                                       int second, 
                                       int expectedSum) {
        Calculator calculator = new Calculator();
        int actualSum = calculator.sum(first, second);
        assertThat(actualSum).isEqualByComparingTo(expectedSum);
    }
}
```